### DISCUSSION TITLE
wiki/help-pages/minecraft-settings/

### DISCUSSION THREAD

**Discussion by giscus** [PrismLauncher/prismlauncher.org#660] [Wiki]
# wiki/help-pages/minecraft-settings/

An Open Source Minecraft launcher with the ability to manage multiple instances, accounts and mods. Focused on user freedom and free redistributability.

https://prismlauncher.org/wiki/help-pages/minecraft-settings/

**Comment by andrpetim** [comment]
And what to do if I don`t have Perfomance tab? 
What to do if I want to change which GPU to use, because i think Nvidium mod doesn`t even work for me
I have two GPU`s: 
Intel(R) HD Graphics 4000
and  NVIDIA GeForce GT 630M
There`s just nothing that changes whenewer i use Nvidium or no

**Comment by Dablob15** [comment]
Minecraft will not launch what can I do

**Comment by vincentf305** [comment]
I don't see a performance tab and I have two GPUs,
